tutorials/misc/index.rst

```diff
@@ -11,3 +11,4 @@ Miscellaneous
    binary_serialization_api
    running_code_in_the_editor
    change_scenes_manually
+   gles2_gles3_differences
```

Comment:
There are a few more things that are probably worth noting:

-GIProbe does not work in GLES2, using lightmaps (BakedLightmap) is recommended.
-Contact shadows also do not work in GLES2
-Reading from depth (DEPTH_TEXTURE) in the shader, does not work on every hardware. On old mobile hardware this is most likely not supported.
-Performance scales poorly in GLES2 with several lights, as each is processed in a separate render pass (in opposition to GLES3 which is all done in a single pass).
-On mobile, GLES2 requires ETC texture compression, while GLES3 requires ETC2. Showing how to enable ETC compression in the project settings if exporting to mobile would be nice.
-Blend Shapes do not work on GLES2 (could be made to work eventually, but it's a lot of work and wont be fast).
-GLES2 besides being linear color, is not HDR capable. This could be noted probably.

Reply:
Thanks reduz! I have added these to the tracked and will submit an updated PR soon